.circleci/config.yml

```diff
@@ -8,14 +8,16 @@ references:
       command: |
         apt-get update -qy
         apt-get install -y git python3-dev python3-pip curl
-        pip3 install coveralls coverage tensorflow
+        pip3 install coveralls coverage tensorflow==1.14
```

Comment:
```suggestion
        pip3 install coveralls coverage tensorflow==1.15
```

Reply:
`1.15` does not work there, it is not within the available versions, do not ask me why

Reply:
btw this won't work with new python 3.8... we have to move to virtualenv also in ci